plugins/main/windows/win-bridge/README.md

```diff
@@ -0,0 +1,27 @@
+# win-bridge plugin
+
+## Overview
+
+With win-bridge plugin, all containers (on the same host) are plugged into an L2Bridge network that has one endpoint in the host namespace.
+
+## Example configuration
+```
+{
+	"name": "mynet",
+	"type": "win-bridge",
+	"ipMasq": true,
+	"clusterNetworkPrefix": "10.244.0.0/16",
```

Comment:
ClusterNetworkPrefix is a pretty runtime (eg Kube) specific term, and it looks like it just gets masqueraded when ipMasq is true.  Could we change the name to "ipMasqNetwork" or something like that?

Also, having both ipMasq and clusterNetworkPrefix is redundant.  We could just have code like:

```
if n.IPMasqNetwork != "" {
    n.ApplyOutboundNatPolicy(n.IPMasqNetwork)
}
```